go.mod

```diff
@@ -82,4 +84,6 @@ require (
 	google.golang.org/api v0.122.0
 	google.golang.org/grpc v1.55.0
 	google.golang.org/protobuf v1.30.0
+	gorm.io/driver/postgres v1.5.2
+	gorm.io/gorm v1.25.2
```

Comment:
we have alot of deps already, let's make this optional behind an interface

Reply:
Take a look ;)